src/models/signature-chain-link.ts

```diff
@@ -0,0 +1,21 @@
+import { prop } from "@typegoose/typegoose";
+import Form from "./form";
+import { SigningPositions } from "@/enums/signing-positions";
+import Position from "./position";
+
+class SignatureChainLink {
```

Comment:
Can we rename this and the `SignatureRequestLink`? I'm thinking maybe this one could be called `GenericSignatureRequest` and the other one could just be named `SignatureRequest`. Makes the relationship between the two more obvious. Also, they're not really "links" anymore anyways

Reply:
yup changed to `Signee` and `GeneriSignee`